yabt/graph_test.py

```diff
@@ -283,11 +283,19 @@ def test_dep_name_typo(basic_conf):
     with pytest.raises(ValueError) as excinfo:
         populate_targets_graph(build_context, basic_conf)
     ex_msg = str(excinfo.value)
+    print(ex_msg)
     assert 'Could not resolve 5 targets' in ex_msg
-    assert ':builderz - buildenv of typo:foo' in ex_msg
-    assert 'typo:bar - seen on command line' in ex_msg
-    assert 'typo:blask - dependency of typo:yapi' in ex_msg
-    assert 'typo:loggin - dependency of typo:base' in ex_msg
-    assert 'typo:zapi - dependency of typo:foo' in ex_msg
+    assert (':builderz (possible misspelling of [\':builder\']) - buildenv of' +
+            ' typo:foo') in ex_msg
+    assert ('typo:bar (possible misspelling of [\'typo:base\', \'typo:yapi\',' +
+            ' \'typo:flask\']) - seen on command line') in ex_msg
+    assert ('typo:blask (possible misspelling of [\'typo:flask\',' +
+            ' \'typo:base\', \'typo:yapi\']) - ' +
+            'dependency of typo:yapi') in ex_msg
+    assert ('typo:loggin (possible misspelling of [\'typo:logging\',' +
+            ' \'typo:foo\', \'typo:yapi\']) - ' +
+            'dependency of typo:base') in ex_msg
+    assert ('typo:zapi (possible misspelling of [\'typo:yapi\', \'typo:base\'' +
+            ', \'typo:flask\']) - dependency of typo:foo') in ex_msg
```

Comment:
add also the case that no target is similar

Reply:
done